confmap/testdata/merge-append-scenarios-featuregate-disabled.yaml

```diff
@@ -1,6 +1,6 @@
 - name: merge-mode-default
   configs:
-    -
+    - |
```

Comment:
I assume this is only needed for the test cases and not in an actual config file?